
# Contains the proximity information of the TOF sensor

Header header           # timestamp in the header is the time the sensor returns the proximity

float32 proximity 		# proximity data [m]

float32 field_of_view   # field of view of sensor [rad]

float32 min_range       # minimum range value [m]

float32 max_range 		# maximum range value [m]
